lib/specter/track.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Specter.TrackLocalStaticSample do
+  @moduledoc """
+  A representation of webrtc.rs TrackLocalStaticSample.
+  """
```

Comment:
Would it make sense to add a little bit of a description of what this is? I see a brief description [here](https://github.com/webrtc-rs/webrtc/blob/master/src/track/track_local/track_local_static_sample.rs#L16-L17), but I'm still not sure what this really represents.

If it doesn't make sense to do that here, I could see us adding something more to the lifecycle guide showing how one would use this in the context of two established peer connections.

Reply:
Sure, I have added a small description. This is how I see track concept at the moment but I can't guarantee it is 100% correct. I think this will be clearer after wrapping a few more modules